CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## [0.3.24](https://github.com/typeorm/typeorm/compare/0.3.23...0.3.24) (2025-05-14)
+
+
+### Bug Fixes
+
+* **capacitor** use query to run PRAGMA statements ([#11467](https://github.com/typeorm/typeorm/issues/11467)) ([d325d9e](https://github.com/typeorm/typeorm/commit/d325d9e63d7936727987b8cc72d756d4e107be4b))
```

Comment:
The commit has a different message, I wonder if this was manually written. But not important to change it.

Reply:
Yeah I manually revised it to make the changelog more clear that this fix is for the capacitor data source type.

The changelog command generated `* capacitor driver PRAGMA bug`